README.md

```diff
@@ -16,17 +34,55 @@ Generated documentation for this SDK can be found at https://rakutentech.github.
 
 To run the example project, clone the repo, and run `pod install` from the Example directory first.
 
-## Requirements
+### Custom parameters
+
+In order to load your MiniApp, you will have to use your own Host App ID and your subscription key. These can eiher be set in project configuration plist (RASApplicationIdentifier and RASProjectSubscriptionKey) or by taping the top right configuration icon in the example application.
```

Comment:
Need to also mention that they should set the base url. On android, we also have a message like this: `We don't currently host a public API, so you will need to provide your own Base URL for API requests.`.

Reply:
Agree with cory + eiher -> either

Reply:
You're right, I base the doc on the custom runtime parameters we had in the application